middlewares/authorizeUsersAndService.ts

```diff
@@ -8,7 +8,7 @@ const { Services } = require("../constants/bot");
 const ROLES = require("../constants/roles");
 const { INTERNAL_SERVER_ERROR_MESSAGE } = require("../constants/progresses");
 
-export const authorization = (allowedRoles: string[], allowedServices: string[]) => {
+export const authorizeAndAuthenticate = (allowedRoles: string[], allowedServices: string[]) => {
```

Comment:
Nit: Wouldn't `authenticateAndAuthorizeUsersAndService` be more meaningful?

Reply:
True, but **authenticateAndAuthorizeUsersAndService** is a bit large, and also the parameters defines the intent of the function

Reply:
`authorizeAndAuthenticate([ROLES.SUPERUSER], [Services.CRON_JOB_HANDLER])`